src/main/resources/META-INF/resources/primefaces/datepicker/0-datepicker.js

```diff
@@ -2488,9 +2495,11 @@
             this._setInitOptionValues();
         },
 
-        updateModel: function (event, value) {
+        updateModel: function (event, value, updateInput) {
             this.value = (value === '' ? null : value);
-            this.inputfield.val(this.getValueToRender());
+            if (updateInput != false) {
```

Comment:
I think you can just say `if (updateInput)` and its truthiness will only enter the block if the value is `true` rather than checking for !false

Reply:
No, not that easy. updateInput is only passed when i want to disable input-update. All other calls to this function are without this parameter.
So we may change this. But this requires all other calls to this function to pass true for the third parameter.